src/Viewer.jsx

```diff
@@ -7,7 +7,7 @@ export const Viewer = ({barColor, formState, language}) => {
 		<>
 			<ThemeProvider theme={typoTheme}>
 			<CssBaseline/>
-			<Grid item xs={12} sm={10} md={5} sx={{border:1, ml:{sm:0,md:4}, mb:{xs:20,sm:20,md:0}, minHeight:{sm:'500px',md:0}, height:(200 + formState.properties.length * 20) + 'px'}} >
+			<Grid container item sx={{border:1}}>
```

Comment:
Can we have a smaller box?, when you have only 1 property the white space is kinda big